yarn-project/constants/src/constants.gen.ts

```diff
@@ -1,4 +1,4 @@
-
+/* eslint-disable */
```

Comment:
Not sure why this wasn't in the previous diff,  the file generator does include this